conf/spgwu.bess

```diff
@@ -169,9 +188,28 @@ class Port:
 
                 # Direct fast path traffic to Frag module
                 merge = __bess_module__("{}Merge".format(name), 'Merge')
-                frag = __bess_module__("{}IP4Frag".format(name), 'IPFrag')
-                merge -> frag:1 -> self.fpo
-                frag:0 -> Sink()
+
+                out = self.fpo
+                # Attach telemeric module (if enabled)
+                if MEASURE == True:
+                   m = Measure(latency_ns_resolution=1, latency_ns_max=100000)
+                   m -> out
+                   out = m
+
```

Comment:
Should we add a checksum here?

Reply:
Not sure I got this. Let's sync up tomorrow to discuss this.. and some other `MEASURE`-related stuff.